src/SingleMovie.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import './SingleMovie.css';
+
+function SingleMovie({selection, setSelection}) {
+    const dummyMovie = {
+        "movie": {
+            id: 1,
+            title: "Fake Movie Title",
+            poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg",
+            backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg",
+            release_date: "2019-12-04",
+            overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!",
+            average_rating: 6,
+            genres: ["Drama"],
+            budget:63000000,
+            revenue:100853753,
+            runtime:139,
+            tagline: "It's a movie!"
+        }
+    }
+
+    const [movie, setMovie] = useState(dummyMovie)
+
+
+
+    //data formatting functions for budget, revenue, runtime, and maybe rating
```

Comment:
Yes, there might be some data cleaning that needs to happen with these fields. My recommendation is to clean the data BEFORE you set it to state. The data in state should be really solid. You don't want missing/bad/unformatted data in state and then just fix it for the DOM.